tarea/task-1-class6-age-form-bootstrap/task-1-class-10-age-form-scripts.js

```diff
@@ -0,0 +1,146 @@
+const $body = document.querySelector('body');
+const $buttonFamilyMembers = document.querySelector('#button-family-members');
+const $resetButton = document.querySelector('#reset-button');
+const $ageFormContent = document.querySelector('#age-form-content');
+const $familyAgeForm = document.querySelector('#family-age-form');
+const $buttonCalculateValues = document.querySelector('#calculate-age-values-button');
+
+$buttonFamilyMembers.onclick = () => {
+	const $familyMembers = document.querySelector('#members-in-family').value;
```

Comment:
This variable is getting the VALUE not a HTML element, so I think it shouldn't have the ``$`` symbol in it